internal/jujuapi/api.go

```diff
@@ -36,9 +36,17 @@ func APIHandler(ctx context.Context, jimm *jimm.JIMM, p Params) http.Handler {
 // ModelHandler creates an http.Handler for "/model" endpoints.
 func ModelHandler(ctx context.Context, jimm *jimm.JIMM, p Params) http.Handler {
 	mux := http.NewServeMux()
-	mux.Handle("/", &jimmhttp.WSHandler{
+	mux.Handle("/{uuid}/api", &jimmhttp.WSHandler{
 		Upgrader: websocketUpgrader,
-		Server:   modelProxyServer{jimm: jimm},
+		Server: &apiProxier{apiServer: apiServer{
+			jimm: jimm,
+		}},
+	})
+	mux.Handle("/{uuid}/", &jimmhttp.WSHandler{
```

Comment:
It's the fallback for any model endpoint which we haven't defined, which is valid behaviour for now. If the handler receives an endpoint it doesn't recognise it will return an appropriate error.

Reply:
i would rather prefer the mux to give a 404 then the wshandler to give back some "cannot upgrade" error.
Don't you think it's best to have the full path here also for clarity and when in the future we need to add new endpoints under this model handler?